src/antsibull_core/utils/io.py

```diff
@@ -71,10 +72,17 @@ async def copy_file(
     flog.debug("Leave")
 
 
-async def write_file(filename: StrOrBytesPath, content: str) -> None:
+async def write_file(
+    filename: StrOrBytesPath, content: str, /, sanitize: bool = False
```

Comment:
Technically speaking, making `filename` and `content` positional-only arguments is a breaking change.